HomeAssistant/AppDelegate.swift

```diff
@@ -25,12 +26,13 @@ let prefs = UserDefaults(suiteName: Constants.AppGroupID)!
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
     var window: UIWindow?
-
+    var safariVC: SFSafariViewController?
     func application(_ application: UIApplication,
                      didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
         Fabric.with([Crashlytics.self])
-        HomeAssistantAPI.sharedInstance.Setup(baseURLString: keychain["baseURL"], password: keychain["apiPassword"],
-                                              deviceID: keychain["deviceID"])
+
+//        HomeAssistantAPI.sharedInstance.setup(baseURLString: keychain["baseURL"], password: keychain["apiPassword"],
```

Comment:
Plz remove any stale lines like this

Reply:
yah. Srry. Taking a clean up pass through it again.